Tools do not compile

### rAthena Hash

16e9d52

### Client Date

Irrelevant

### Server Mode

Pre-Renewal, Renewal

### Result

Tools do not compile because commit cad5cab976290283bf47738aa8a383891e33b82a introduced reference to `gettick` in battle.hpp. 

### Relevant Log Output

```Shell
[build] /usr/bin/ld: CMakeFiles/yaml2sql.dir/yaml2sql.cpp.o: in function `battle_fix_damage(block_list*, block_list*, long, long, unsigned short)':
[build] /home/secret/rathena/src/map/battle.hpp:128:(.text+0x24): undefined reference to `gettick()'
[build] /usr/bin/ld: /home/secret/rathena/src/map/battle.hpp:128:(.text+0x5a): undefined reference to `battle_damage(block_list*, block_list*, long, long, unsigned short, unsigned short, damage_lv, unsigned short, bool, long, bool)'
[build] /usr/bin/ld: CMakeFiles/yamlupgrade.dir/yamlupgrade.cpp.o: in function `battle_fix_damage(block_list*, block_list*, long, long, unsigned short)':
[build] /home/secret/rathena/src/map/battle.hpp:128:(.text+0x2c): undefined reference to `gettick()'
[build] /usr/bin/ld: /home/secret/rathena/src/map/battle.hpp:128:(.text+0x62): undefined reference to `battle_damage(block_list*, block_list*, long, long, unsigned short, unsigned short, damage_lv, unsigned short, bool, long, bool)'
[build] collect2: error: ld returned 1 exit status
[build] /usr/bin/ld: gmake[2]: *** [src/tool/CMakeFiles/yaml2sql.dir/build.make:183: /home/secret/rathena/yaml2sql] Error 1
[build] gmake[1]: *** [CMakeFiles/Makefile2:803: src/tool/CMakeFiles/yaml2sql.dir/all] Error 2
[build] gmake[1]: *** Waiting for unfinished jobs....
[build] collect2: error: ld returned 1 exit status
[build] gmake[2]: *** [src/tool/CMakeFiles/yamlupgrade.dir/build.make:183: /home/secret/rathena/yamlupgrade] Error 1
[build] gmake[1]: *** [CMakeFiles/Makefile2:831: src/tool/CMakeFiles/yamlupgrade.dir/all] Error 2
[build] CMakeFiles/csv2yaml.dir/csv2yaml.cpp.o: in function `battle_fix_damage(block_list*, block_list*, long, long, unsigned short)':
[build] /home/secret/rathena/src/map/battle.hpp:128:(.text+0x48): undefined reference to `gettick()'
[build] /usr/bin/ld: /home/secret/rathena/src/map/battle.hpp:128:(.text+0x7e): undefined reference to `battle_damage(block_list*, block_list*, long, long, unsigned short, unsigned short, damage_lv, unsigned short, bool, long, bool)'
[build] collect2: error: ld returned 1 exit status
[build] gmake[2]: *** [src/tool/CMakeFiles/csv2yaml.dir/build.make:183: /home/secret/rathena/csv2yaml] Error 1
[build] gmake[1]: *** [CMakeFiles/Makefile2:775: src/tool/CMakeFiles/csv2yaml.dir/all] Error 2
[build] gmake: *** [Makefile:136: all] Error 2
[proc] The command: /usr/bin/cmake --build /home/secret/rathena/build --config Debug --target all -j 26 -- exited with code: 2
[driver] Build completed: 00:00:00.356
[build] Build finished with exit code 2
```


### Expected Result

Tools compile.

### How to Reproduce

Build on Linux with CMake

### Official Information

N/A

### Modifications that may affect results

_No response_

--- a/src/map/battle.cpp
+++ b/src/map/battle.cpp
@@ -422,6 +422,10 @@ int battle_delay_damage(t_tick tick, int amotion, struct block_list *src, struct
 	return 0;
 }
 
+int battle_fix_damage(struct block_list* src, struct block_list* target, int64 damage, t_tick walkdelay, uint16 skill_id) {
+	return battle_damage(src, target, damage, walkdelay, 0, skill_id, ATK_DEF, BF_MISC, false, gettick(), false);
+}
+
 /**
  * Does attribute fix modifiers.
  * Added passing of the chars so that the status changes can affect it. [Skotlex]

--- a/src/map/battle.hpp
+++ b/src/map/battle.hpp
@@ -124,9 +124,7 @@ int64 battle_calc_pk_damage(block_list &src, block_list &bl, int64 damage, uint1
 
 int battle_damage(struct block_list *src, struct block_list *target, int64 damage, t_tick delay, uint16 skill_lv, uint16 skill_id, enum damage_lv dmg_lv, unsigned short attack_type, bool additional_effects, t_tick tick, bool spdamage);
 int battle_delay_damage (t_tick tick, int amotion, struct block_list *src, struct block_list *target, int attack_type, uint16 skill_id, uint16 skill_lv, int64 damage, enum damage_lv dmg_lv, t_tick ddelay, bool additional_effects, bool spdamage);
-static int battle_fix_damage(struct block_list* src, struct block_list* target, int64 damage, t_tick walkdelay, uint16 skill_id) {
-	return battle_damage(src, target, damage, walkdelay, 0, skill_id, ATK_DEF, BF_MISC, false, gettick(), false);
-}
+int battle_fix_damage(struct block_list* src, struct block_list* target, int64 damage, t_tick walkdelay, uint16 skill_id);
 
 int battle_calc_chorusbonus(map_session_data *sd);
 

